sdk/communication/azure-communication-callingserver/azure/communication/callingserver/__init__.py

```diff
@@ -55,6 +56,7 @@
     'AnswerCallResult',
     'AudioRoutingGroupResult',
     'CreateAudioRoutingGroupResult',
-    'TransferCallResult'
+    'TransferCallResult',
+    'ParticipantsUpdatedEvent'
```

Comment:
exposing ParticipantsUpdatedEvent as it is exposed as an event type enum and it is needed for the sample